test/sequential/test-make-doc.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+// This tests that `make doc` generates the documentation properly.
```

Comment:
Nits: as per https://github.com/nodejs/node/blob/ff747e3fe8fc960bb6cd04185167bd6376940998/doc/guides/writing-tests.md#L26-L32 headers should be
```js
'use strict';
const common = require('../common');
if (common.isWindows) {
  common.skip('Do not make doc on Windows');
}

// This tests that `make doc` generates the documentation properly.
// Note that for this test to pass, `make doc` must be run first.
 
const assert = require('assert');
const fs = require('fs');
const path = require('path');
```